website/dbt-versions.js

```diff
@@ -22,6 +22,14 @@ exports.versions = [
 ]
 
 exports.versionedPages = [
+  {
+    "page": "/reference/resource-configs/target_database",
+    "lastVersion": "1.8",
+  },
+  {
+    "page": "/reference/resource-configs/target_schema",
```

Comment:
Looks like our version control here isn't working. Since the original idea was to remove or add a warning, I combined them, but as-is these pages will still appear, they'll just have a note at the top with links to the new configs

Reply:
> these pages will still appear, they'll just have a note at the top with links to the new configs

I like that!